cmd/example/docs/generated/service.html

```diff
@@ -1 +1 @@
-<!DOCTYPE html><html lang=en><head><title>Animal Bars</title><meta name='generator' content='github.com/lonnblad/go-service-doc'><link rel="stylesheet" href="/docs/service/markdown.css"></head><body class="markdown-body"><div class="flex-container"><div class="menu-container"><div  class=menu-header><h1>Animal Bars</h1><form class=menu-search action="/docs/service/search" method="get"><input type="text" placeholder="Search.." name="q" value="" onfocus="var temp_value=this.value; this.value=''; this.value=temp_value" autofocus /><button type="submit">Search</button></form></div><ul><li><a href="/docs/service#animal_bars">Animal Bars</a></li><li><a href="/docs/service/donkey-bar#donkey">Donkey Bar</a><ul><li><a href="/docs/service/donkey-bar#code_examples">Code Examples</a></li></ul></li><li><a href="/docs/service/monkey-bar#monkey">Monkey Bar</a><ul><li><a href="/docs/service/monkey-bar#code_examples">Code Examples</a></li></ul></li></ul></div><div class="doc-container"><h1 id="animal_bars">Animal Bars</h1><p><img src="/docs/service/static/animal-bars.svg" alt="The bars" /></p></div></div></body></html>
\ No newline at end of file
+<!DOCTYPE html><html lang=en><head><title>Animal Bars</title><meta name='generator' content='github.com/lonnblad/go-service-doc'><link rel="stylesheet" href="/bars/markdown.css"></head><body class="markdown-body"><div class="flex-container"><div class="menu-container"><div  class=menu-header><h1>Animal Bars</h1><form class=menu-search action="/bars/search" method="get"><input type="text" placeholder="Search.." name="q" value="" onfocus="var temp_value=this.value; this.value=''; this.value=temp_value" autofocus /><button type="submit">Search</button></form></div><ul><li><a href="/bars#animal_bars">Animal Bars</a></li><li><a href="/bars/donkey-bar#donkey">Donkey Bar</a></li><li><a href="/bars/monkey-bar#monkey">Monkey Bar</a><ul><li><a href="/bars/monkey-bar#code_examples">Code Examples</a></li></ul></li></ul></div><div class="doc-container"><h1 id="animal_bars">Animal Bars</h1><p><img src="/bars/static/animal-bars.svg" alt="The bars" /></p></div></div></body></html>
```

Comment:
It looks like you have renamed `cmd/example/docs/src/static/animal-bars.svg` to `cmd/example/docs/src/static/bars.svg`.

The generated `service.html` and `docs.go` both refers to the file as: `/bars/static/animal-bars.svg`

Reply:
good catch, thanks!

Reply:
You missed one in `service.html`

Reply:
fixed